### ISSUE TITLE
Account Billing 

### ISSUE THREAD

**Issue by mlande5** [jackbayliss/Mission-Chief-Bot#66]
What the heck is up with continuously having to pay to use the program. The site says once a month but if you look at my account I've been billed multiple times and haven't even done 30 days. Why?

**Comment by mlande5** [comment]
Your plan renews on June 14, 2022. Your plan renews on June 12, 2022. Your plan renews on June 4, 2022.

May 14, 2022 | £1.00 | Paid | MissionChiefBot Access
May 12, 2022 | £1.00 | Paid | MissionChiefBot Access
May 4, 2022 | £1.00 | Paid | MissionChiefBot Access
May 14, 2022](https://invoice.stripe.com/i/acct_1G8VSLG4CfxIXOU3/live_YWNjdF8xRzhWU0xHNENmeElYT1UzLF9MZ21mTVBmbVpwa2Vxc25VdDJsM2huaWk0cmVmNEI2LDQzMDkxNzI20200KkrNIYaY?s=il?locale=en-US)
£1.00
Paid
MissionChiefBot Access
[May 12, 2022](https://invoice.stripe.com/i/acct_1G8VSLG4CfxIXOU3/live_YWNjdF8xRzhWU0xHNENmeElYT1UzLF9MZnlMelhHRjBVOG9JbGNrWjVrV2tXQjM1eTQxY3hYLDQzMDkxNzI20200ZyOqtRaV?s=il?locale=en-US)
£1.00
Paid
MissionChiefBot Access
[May 4, 2022](https://invoice.stripe.com/i/acct_1G8VSLG4CfxIXOU3/live_YWNjdF8xRzhWU0xHNENmeElYT1UzLF9MZDYzWEdZT3AxZHBSZE5ybXBYSVJYODlBUVU2MVlPLDQzMDkxNzI20200DZA9KbCs?s=il?locale=en-US)
£1.00
Paid
MissionChiefBot Access

**Comment by jackbayliss** [comment]
Sorry you're having issues,This is basically stripes doing- if you email me your email I can take a look for you On 14 May 2022 18:54, mlande5 ***@***.***> wrote:
What the heck is up with continuously having to pay to use the program. The site says once a month but if you look at my account I've been billed multiple times and haven't even done 30 days. Why?

**Comment by mlande5** [comment]
Hi, 
I think maybe I have figured out part of the issue. I received notice that my May 4 account charge was denied. I added a new card to my account. Apparently that didn't make any difference as the default card was the one that had been cancelled because of detected fraud by the bank. 
I just received another notice that my monthly subscription couldn't be completed as the charge was denied even though there was a valid card on file but it wasn't the default card. 
So, when I went back into my account I saw that my previous card was still the default. I just changed the default and deleted the card that should stop the issue. 
I'm enjoying the software program and feel that the issue has been resolved and will wait and see if there is any further issue. I hope you are good with this? 
Thank you 
***@***.*** 
    On Saturday, May 14, 2022, 11:02:42 AM PDT, Jack ***@***.***> wrote:  
 
 

Sorry you're having issues,This is basically stripes doing- if you email me your email I can take a look for you On 14 May 2022 18:54, mlande5 ***@***.***> wrote:
What the heck is up with continuously having to pay to use the program. The site says once a month but if you look at my account I've been billed multiple times and haven't even done 30 days. Why?
